docs/onboard.md

```diff
@@ -0,0 +1,86 @@
+# Onboard
+
+## Vendor
+
+### Step 1
+Send in your request for a TechPass account to TechPass Team
+
+Once your request has been approved, you will receive an username via email and initial passcode via SMS
+![Vendor_Email_Invitation](assets/support/Vendor_Email_Invitation.jpeg)
+![Vendor_SMS_1](assets/support/Vendor_SMS_1.jpeg)
+
+
+### Step 2
+Sign in to TechPass Portal with username and initial passcode
+![Vendor_email](assets/support/Vendor_email.jpeg)
+![Vendor_InitialPassword](assets/support/Vendor_InitialPassword.jpeg)
+
+### Step 3
+Install the Microsoft Authenticator app on your new mobile device by following the steps in the [Download and install the Microsoft Authenticator app](https://docs.microsoft.com/en-us/azure/active-directory/user-help/user-help-auth-app-download-install) article.
+
+### Step 4
+Follow the series of prompt to set up MFA in your Microsoft Authenticator app using your mobile device
+![Vendor_MFA_1](assets/support/Vendor_MFA_1.png)
+![Vendor_MFA_2](assets/support/Vendor_MFA_2.png)
+![Vendor_MFA_3](assets/support/Vendor_MFA_3.jpeg)
+![Vendor_MFA_4](assets/support/Vendor_MFA_4.png)
+![Vendor_MFA_5](assets/support/Vendor_MFA_5.png)
+![Vendor_MFA_6](assets/support/Vendor_MFA_6.jpeg)
+![Vendor_MFA_7](assets/support/Vendor_MFA_7.jpeg)
+![Vendor_MFA_8](assets/support/Vendor_MFA_8.jpeg)
+![Vendor_MFA_9](assets/support/Vendor_MFA_9.jpeg)
+
+### Step 5
+Provide your initial passcode, add new password and confirm your password
+![Vendor_UpdatePassword](assets/support/Vendor_UpdatePassword.jpeg)
+
+### Step 6
+Pls read, understand and accept terms of use to proceed with the TechPass portal set up
+![Vendor_TermsOfUse](assets/support/Vendor_TermsOfUse.jpeg)
+![Vendor_TermsOfPolicy](assets/support/Vendor_TermsOfPolicy.png)
+
+### Step 7
+TechPass portal will be opened and you will see My Profile section
+![Vendor_MyProfile](assets/support/Vendor_MyProfile.jpeg)
+
+## Public Officer
+
+### Step 1
+Send in your request for a TechPass account to TechPass Team
```

Comment:
public officers will do self sign up via gsib machine.
you can update this instruction in the next pr